src/index.js

```diff
@@ -18,7 +18,8 @@ export { default as Icon } from './Icon'
 export { default as IconButton } from './IconButton'
 export { default as Image } from './Image'
 export { default as Input } from './Input'
-export { default as InputField } from './InputField'
+export { default as InputField } from './FormField'
+export { default as FormField } from './FormField'
```

Comment:
:+1: keeps the current API intact

Reply:
@jxnblk  Do you think we should plan on deprecating `InputField` part of v2?